db/migrate/20240912193815_remove_relationship_from_parents.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class RemoveRelationshipFromParents < ActiveRecord::Migration[7.2]
+  def change
+    change_table :parents, bulk: true do |t|
+      t.remove :relationship, type: :integer
+      t.remove :relationship_other, type: :string
```

Comment:
These are harmless but I don't think they do anything:

```suggestion
      t.remove :relationship
      t.remove :relationship_other
```

Reply:
Oh fair